README.md

```diff
@@ -1 +1,151 @@
-# mpd-parser [![Build Status](https://travis-ci.org/forbesjo/mpd-parser.svg?branch=master)](https://travis-ci.org/forbesjo/mpd-parser)
+# mpd-parser
+
+mpd parser
+
+## Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Installation](#installation)
+- [Usage](#usage)
+  - [Parsed Output](#parsed-output)
+- [Including the Parser](#including-the-parser)
+  - [`<script>` Tag](#script-tag)
+  - [Browserify](#browserify)
+  - [RequireJS/AMD](#requirejsamd)
+- [License](#license)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Installation
+
+```sh
+npm install --save mpd-parser
+```
+
+The npm installation is preferred, but Bower works, too.
+
+```sh
+bower install  --save mpd-parser
+```
+
+## Usage
+
+```js
+var manifest = [
+  '<>',
+  '<>',
+].join('\n');
+
+var parsedManifest = mpdParser.parse(manifest);
+```
+
+### Parsed Output
+
+The parser ouputs a plain javascript object with the following structure:
+
+```js
+Manifest {
```

Comment:
The types here can be expanded upon more. Define what Playlist and Segment are, what properties are optional. Maybe jsdoc can do this automatically from the source?